docs/TROUBLESHOOTING.md

```diff
@@ -91,4 +91,11 @@ Error: Error applying plan:
 	* aws_apigatewayv2_domain_name.websockets_domain: error creating API Gateway v2 domain name: BadRequestException: The provided certificate does not exist.
 ```
 
-When this happens, re-run the deploy.
\ No newline at end of file
+When this happens, re-run the deploy.
+
+
+### aws-xray-sdk version 3.1.0
```

Comment:
Is this a known issue in that library? Do we have a Trello card to follow up on investigating/revisiting this?

Reply:
We have a Trello card https://trello.com/c/yniSYUyL